test/jdk/java/util/Currency/ValidateISO4217.java

```diff
@@ -267,20 +272,21 @@ private static List<Arguments> additionalCodesProvider() {
      * throws an IllegalArgumentException or returns null. The test data
      * supplied is every possible combination of AA -> ZZ.
      */
-    @ParameterizedTest
-    @MethodSource("codeCombos")
-    public void twoLetterCodesTest(String country) {
-        if (codes[toIndex(country)] == UNDEFINED) {
-            // if a code is undefined / 0, creating a Currency from it
-            // should throw an IllegalArgumentException
-            assertThrows(IllegalArgumentException.class,
-                    ()-> Currency.getInstance(Locale.of("", country)),
-                    "Error: This should be an undefined code and throw IllegalArgumentException: " + country);
-        } else if (codes[toIndex(country)] == SKIPPED) {
-            // if a code is marked as skipped / 2, creating a Currency from it
-            // should return null
-            assertNull(Currency.getInstance(Locale.of("", country)),
-                    "Error: Currency.getInstance() for this locale should return null: " + country);
+    @Test
+    public void twoLetterCodesTest() {
+        for (String country : codeCombos()) {
+            if (codes[toIndex(country)] == UNDEFINED) {
+                // if a code is undefined / 0, creating a Currency from it
+                // should throw an IllegalArgumentException
+                assertThrows(IllegalArgumentException.class,
+                        () -> Currency.getInstance(Locale.of("", country)),
+                        "Error: This should be an undefined code and throw IllegalArgumentException: " + country);
+            } else if (codes[toIndex(country)] == SKIPPED) {
+                // if a code is marked as skipped / 2, creating a Currency from it
+                // should return null
+                assertNull(Currency.getInstance(Locale.of("", country)),
+                        "Error: Currency.getInstance() for this locale should return null: " + country);
+            }
```

Comment:
The previous output was overflowing due to the method being a `@ParameterizedTest` since it was testing against 26*26 inputs. Changing to `@Test` makes diagnostics easier, as there is no more overflow.